src/OSPSuite.UI/Views/Importer/ColumnMappingControl.cs

```diff
@@ -52,13 +56,14 @@ public ColumnMappingControl(IImageListRetriever imageListRetriever)
          _gridViewBinder = new GridViewBinder<ColumnMappingDTO>(columnMappingGridView);
          columnMappingGridView.OptionsView.ShowGroupPanel = false;
          columnMappingGridView.OptionsMenu.EnableColumnMenu = false;
-         columnMappingGridView.CellValueChanged += (s, e) => _presenter.ValidateMapping();
+         columnMappingGridView.MultiSelect = false;
+         columnMappingGridView.CellValueChanged += (o, e) => OnEvent(_presenter.ValidateMapping);
 
          columnMappingGridView.OptionsView.ShowButtonMode = ShowButtonModeEnum.ShowOnlyInEditor;
-         columnMappingGridView.OptionsBehavior.EditorShowMode = EditorShowMode.MouseUp;
-         columnMappingGridView.MouseDown += onMouseDown;
-         columnMappingGrid.ToolTipController = new ToolTipController();
-         columnMappingGrid.ToolTipController.GetActiveObjectInfo += (o, e) => this.DoWithinExceptionHandler(() => onGetActiveObjectInfo(o, e));
+         columnMappingGridView.OptionsBehavior.EditorShowMode = EditorShowMode.MouseDown;
+         columnMappingGridView.MouseDown += (o, e) => OnEvent(onMouseDown, o, e);
```

Comment:
EVery single event in the UI should be in OnEvent. Otherwise, crash would lead to app disappearing completely. Major

Reply:
done overall for the events in the Importer/Views